ajax/libs/snarkdown/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "snarkdown",
+  "description": "Transform Markdown into HTML.",
+  "authors": [
+    "Jason Miller <jason@developit.ca>"
+  ],
```

Comment:
Let's modify from A to B. Because there is one author not authors.
- A
```
  "authors": [
    "Jason Miller <jason@developit.ca>"
  ],
```

- B
```
  "author": "Jason Miller <jason@developit.ca>",
```

Thank you.